docs/snap-clone.md

```diff
@@ -42,7 +48,9 @@
 Also PVC cannot be deleted if it's having snapshots. Make sure all the snapshots
 on the PVC are deleted before you delete the PVC.**
 
-## Snapshot
+## Create CephFS snapshot and Clone Volume
+
+### Snapshot
```

Comment:
Is this still needed? You improved the name of the title, so I think this can be removed now.

Reply:
I guess its better to have it as mentioned in the contents. Let me know if you still want me to remove it. I will modify the content too.